jboss-eap-standalone-openshift/azuredeploy.json

```diff
@@ -267,15 +225,15 @@
       }
     },
     {
-      "apiVersion": "[variables('apiVersionCompute')]",
       "type": "Microsoft.Compute/virtualMachines/extensions",
       "name": "[concat(resourceGroup().name,'/installcustomscript')]",
+      "apiVersion": "2017-03-30",
       "location": "[resourceGroup().location]",
```

Comment:
you need to go through all the resources and ensure the location properties are set to [parameters('location')]

Reply:
This is line 231 now. I changed all the location properties to "[parameters('location')]" throughout the template. However, for this extensions resource, it didn't work. I have no idea why. I works with "[resourceGroup().location]", however.